tests/console/python_scientific.pm

```diff
@@ -37,6 +37,8 @@ sub run_python_script {
 sub run {
     select_serial_terminal;
 
+    return if (check_var('PHUB_READY', 'no'));
```

Comment:
Maybe if you extract the variable to either `registration` we can do:

```suggestion
    return if is_phub_ready();
```

which is cleaner :)

Reply:
You are correct, let me do that